lib/solarwinds_apm/config.rb

```diff
@@ -75,6 +76,28 @@ def self.set_log_level
       SolarWindsAPM.logger.level = debug_level < 0 ? 6 : [4 - debug_level, 0].max
     end
 
+    def self.enable_disable_config(env_var, key, value)
+      default_value = :enabled # Set a default value
```

Comment:
It's incorrect to assume default_value for all enable/disable options (esp. in the future) to be :enabled.  It would also be good if this `enable_disable_config` can be made generic to resolve options that can come from both env var and config file, such as tag sql.

Reply:
Refactored further that more generic to take either :enabled/:disabled or true/false. The default value now is a parameter that we can define easily.